src/sonic-config-engine/tests/test_j2files.py

```diff
@@ -295,6 +295,11 @@ def test_qos_dscp_remapping_render_template(self):
             'qos-arista7260-dualtor.json',
             'qos-arista7260-t1.json'
         ]
+        config_bcm_sample_outputs = [
```

Comment:
I suggest adding another test case to cover the bcm.config change

Reply:
added